scripts/local-cluster.sh

```diff
@@ -248,6 +250,8 @@ resolve() {
 }
 
 config() {
+  : "${CK8S_PGP_FP:?Missing CK8S_PGP_FP}"
+
```

Comment:
Nit: For consistency it would be nice if it used the same style as in the rest of the script.

Reply:
Ah, yeah I missed that this script was checking things in a different way. Do you know why this script does not use the same style as the bin scripts (i.e. the style I tried to add in this PR)?
I have now updated to reflect the usage in this script.

Reply:
I've done so in all scripts I've written since I think it improves the experience, as the error isn't managed in a separate way compared to how we check for other errors or missing things.
And the bin script has been in a constant "will eventually be replaced" so I assume that is why there's never been big improvements for those.